### DISCUSSION TITLE
Localhost not working

### DISCUSSION THREAD

**Discussion by WoolEndy** [danny-avila/LibreChat#270] [Troubleshooting] [answered]
Localhost doesn't work when running the program. This pops up: 
<img width="1128" alt="image" src="https://github.com/danny-avila/chatgpt-clone/assets/111644240/8382bc45-5d4d-4566-898b-75af87653b98">

When exiting the program, this is what it says:
<img width="867" alt="image" src="https://github.com/danny-avila/chatgpt-clone/assets/111644240/18ca04ad-ab6e-40ce-8200-d8d0a3bb6233">

**Comment by danny-avila** [comment]
There could be a number of issues why, mostly unrelated to the project but related to your local setup. One user had the same issue and we helped them on discord with the following resources:

https://bobbyhadz.com/blog/javascript-crbug-1173575-non-js-module-files-deprecated

https://stackoverflow.com/questions/67191286/crbug-1173575-non-js-module-files-deprecated-chromewebdata-index%ea%9e%89530595551

In the end, it was due to a chrome plugin as it would work in Incognito mode. Could you try that first and then maybe go through the links I sent?

**Comment by fuegovic** [comment]
The only thing that this error tell is that it wasn't started when you tried to close it. As to why, there could be a lot of reasons.

Try to run the command `npm ci` from the root dir:

Go in your install directory (e.g. C:\chatgpt-clone ) 
Right click -> open in terminal (or something similar)
enter the command:  npm ci

**Reply by WoolEndy** [reply]
I found this error, but I did install the module.
<img width="535" alt="image" src="https://github.com/danny-avila/chatgpt-clone/assets/111644240/9a1d42ec-6d99-41db-97fd-cface56fa977">

**Reply by WoolEndy** [reply, marked as answer]
I got it working! Ironically, I asked GPT-4 for the solution, and it told me to try
```
npm install express
```
in C:\chatgpt-clone\api, so that worked.
